src/engine/sim_engine/engine.rs

```diff
@@ -314,5 +314,4 @@ mod tests {
             Ok(RenameAction::NoSource)
         );
     }
-
```

Comment:
Please remove stray line removal.

Reply:
A reason behind undoing this change?

Reply:
It's the only line you touched in ```sim_engine/engine.rs```, because there were no changes needed for the sim engine. It would be better if the file just wasn't part of the PR at all.

Reply:
Ah indeed. Might have changed when looking around...